CMakeLists.txt

```diff
@@ -173,6 +173,25 @@ if(USE_NGRAPH AND NOT MSVC)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-comment")
   set(NGRAPH_INSTALL_PREFIX ${CMAKE_BINARY_DIR}/3rdparty/ngraph-mxnet-bridge)
 
+  # Flags required for SDL325
```

Comment:
Is "SDL 325" meaningful outside of Intel?

Reply:
No, but it's meaningful to us. Do you prefer an explanation (e.g. compile with defenses enabled) instead?